lib/jsonapi/filtering.rb

```diff
@@ -17,6 +17,9 @@ def self.extract_attributes_and_predicates(requested_field)
           .detect_and_strip_from_string!(field_name)
         predicates << Ransack::Predicate.named(predicate)
       end
+      unless predicates.present? || Ransack.options[:ignore_unknown_conditions]
+        raise ArgumentError, "No valid predicates for #{requested_field}"
+      end
```

Comment:
Hmm, I think this is handled already by Ransack:
https://github.com/stas/jsonapi.rb/blob/master/lib/jsonapi/patches.rb#L6

Just reset the option in your config and it should work, no?

Reply:
@stas: It seems to be ignored, from my testing.